app1/admin.py

```diff
@@ -1,3 +1,19 @@
-from django.contrib import admin
+#from django.contrib import admin
 
 # Register your models here.
+from django.contrib import admin
+from .models import YourModel  # Import your model(s) here
+
+@admin.register(YourModel)
```

Comment:
**Suggestion**: Ensure to replace 'YourModel' with the actual model name being registered in the admin interface for clarity and consistency. <br> **Code Suggestion**: 
 ```
 +@admin.register(YourModel)
 ```